src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -79,225 +126,477 @@ const useStyles = makeStyles()(() => ({
-            flexDirection="column"
-            style={{ paddingLeft: theme.spacing(1) }}
-          >
-            <Box className={classes.buttonTop}>
-              <Button
-                variant="contained"
-                data-testid="confirmButton"
-                style={{ width: '100%' }}
-                onClick={() =>
-                  updateFunction(
-                    id,
-                    true,
-                    values.statusValue,
-                    values.amountCurrency,
-                    parseFloat(`${values.amount}`),
-                    values.frequencyValue,
-                  )
-                }
-              >
-                {t('Confirm')}
-              </Button>
-            </Box>
-            <Box className={classes.buttonBottom}>
-              <Button
-                variant="contained"
-                style={{ width: '100%' }}
-                data-testid="doNotChangeButton"
-                onClick={() => updateFunction(id, false)}
-              >
-                {"Don't Change"}
-              </Button>
-            </Box>
-            <Box>
-              <IconButton
-                data-testid="goToContactsButton"
-                onClick={() =>
-                  push({
-                    pathname: `/accountLists/[accountListId]/contacts/[contactId]`,
-                    query: { accountListId, contactId: id },
-                  })
-                }
-              >
-                <SearchIcon />
-              </IconButton>
-              <IconButton
-                data-testid="hideButton"
-                onClick={() => hideFunction(id)}
-              >
-                <VisibilityOffIcon />
-              </IconButton>
-            </Box>
-          </Box>
-        </Grid>
-      </Grid>
+      <Formik
+        initialValues={{
+          statusValue: statusValue,
+          pledgeCurrency: amountCurrency,
+          pledgeAmount: amount || suggestedAmount,
+          pledgeFrequency:
+            (frequencyValue as PledgeFrequencyEnum) || suggestedFrequency,
+        }}
+        validationSchema={commitmentInfoFormSchema}
+        onSubmit={async (values) => {
+          await onSubmit(values);
+        }}
+      >
+        {({
+          values: {
+            statusValue,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          },
+          handleSubmit,
+          setFieldValue,
+          errors,
+        }): ReactElement => {
+          const modalContact = {
+            id: id,
+            status: statusValue,
+            name: name,
+            pledgeCurrency,
+            pledgeAmount,
+            pledgeFrequency,
+          };
+          return (
+            <Form onSubmit={handleSubmit}>
+              <Grid container className={classes.formWrapper}>
+                <Card className={classes.formInner}>
+                  <Grid container>
+                    <Grid item lg={4} md={4} xs={12}>
```

Comment:
I think the `lg` is redundant. Unless the `md` is. One of them should be able to be removed though.